src/Administration/Resources/app/administration/src/module/sw-order/view/sw-order-create-initial/index.js

```diff
@@ -12,13 +12,22 @@ export default {
 
     compatConfig: Shopware.compatConfig,
 
+    inject: ['repositoryFactory'],
+
     created() {
         this.createdComponent();
     },
 
     methods: {
         createdComponent() {
-            const { customer } = this.$route.params;
+            const customerRepository = this.repositoryFactory.create('customer');
```

Comment:
Please move the repository creation to a computed to streamline it with all other components